doc/source/security/encrypt-traffic.rst

```diff
@@ -12,6 +12,52 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encryption.
+By default, it is disabled, meaning that you need to configure SSL manually
```

Comment:
"By default ...." statement contradict presence of the said variable.
So this statement (though existing) in new context should be changed.